pkg/model/volumetype.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2017 Huawei Technologies Co., Ltd. All Rights Reserved.
```

Comment:
The voume type is only used in cinder driver, it's not very good to put volumetype.go file in pkg/model directory. Puting it into cinder.go file maybe better.